src/msg/async/ProtocolV2.cc

```diff
@@ -1580,6 +1585,16 @@ CtPtr ProtocolV2::throttle_dispatch_queue() {
           << connection->dispatch_queue->dispatch_throttler.get_current() << "/"
           << connection->dispatch_queue->dispatch_throttler.get_max()
           << " failed, just wait." << dendl;
+      ceph::mono_time throttle_now = ceph::mono_clock::now();
+      auto duration = std::chrono::duration_cast<std::chrono::seconds>(throttle_now - throttle_prev);
+      if (duration >= msgr->cct->_conf.get_val<std::chrono::seconds>("ms_dispatch_throttle_log_interval")) {
```

Comment:
Getting config options can be expensive. Have the `MonClient` monitor this option for changes too and configure the messenger.